hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/FqnResolver.java

```diff
@@ -0,0 +1,7 @@
+package org.hypertrace.traceenricher.enrichment.enrichers.resolver.backend;
+
+import org.hypertrace.core.datamodel.Event;
+
+public interface FqnResolver {
+  String resolve(String hostname, Event event);
```

Comment:
Having some docs would be useful

Reply:
Just a comment about FQN resolver should help.